packages/web/src/components/edit/fields/MultiTrackSidebar.tsx

```diff
@@ -16,10 +16,13 @@ import { isEmpty } from 'lodash'
 import DynamicImage from 'components/dynamic-image/DynamicImage'
 import layoutStyles from 'components/layout/layout.module.css'
 
-import { UploadFormScrollContext } from '../UploadPage'
-import { useIndexedField } from '../hooks'
-import { SingleTrackEditValues, TrackEditFormValues } from '../types'
-import { UploadPreviewContext } from '../utils/uploadPreviewContext'
+import { UploadFormScrollContext } from '../../../pages/upload-page/UploadPage'
```

Comment:
a few cases of absolute imports, and others that are relative, just wondering if its worth being consistent